In [15]:
import numpy as np
from scipy import stats
from scipy.stats import norm


# Q.5

If the sample were based on 2,000 users, could Microsoft conclude that Mozilla has a less than 5% share of the market


Ans:

In [16]:
# Define the sample size, sample proportion, and hypothesized population proportion
n = 2000 #sample size
p = 0.046 #sample_prop 4.6% expressed as a decimal
p_null = 0.05 #5% expressed as a decimal


In [18]:
# Calculate the z-score
#z = (sample_prop - null_prop) / math.sqrt(null_prop * (1 - null_prop) / n)
z = (0.046 - 0.05) / np.sqrt(0.05 * (1 - 0.05) / 2000)
print(z)

-0.820782681668124


Therefore,  z-test statistics is -0.821.
Now, the level of significance at 5 percent, the z table will give the critical value of -1.96 to the left-tailed test.

-0.820 > -1.96 ;

Thus we don't have sufficient evidence to reject the null hypothesis 

So we can conclude that the Mozilla has equal to 5%  or more than 5 % share of the market.